### PR TITLE
Addressing issues in _plot_html

### PR DESCRIPTION
**Pull request by yankev**
#504 #505

### CODE REVIEW

**Comment by yankev** [comment]
@BRONSOLO @theengineear

File: plotly/offline/offline.py [hunk 1]
```diff
@@ -176,11 +176,11 @@ def _plot_html(figure_or_data, show_link, link_text, validate,
         width = str(width) + 'px'
 
     try:
-        float(width)
+        float(height)
     except (ValueError, TypeError):
         pass
     else:
-        width = str(width) + 'px'
+        height = str(height) + 'px'
```
Comment (theengineear) [inline plotly/offline/offline.py]: 😛

**Comment by theengineear** [comment]
💃

**Comment by cldougl** [comment]
@yankev wanna update the CHANGELOG and add this before I update on pip this afternoon?

**Pull request merged**